packages/tailwindcss/src/important.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, test } from 'vitest'
+import { compile } from '.'
+
+const css = String.raw
+
+test('Utilities can be wrapped in a selector', async () => {
+  // This is the v4 equivalent of `important: "#app"` from v3
+  let input = css`
+    @import 'tailwindcss/utilities' selector(#app);
```

Comment:
Was this the final syntax we wanted? I'm fine with it, but it's not super clear what this does (could be a documentation problem).

But it's almost like you want:

If you are talking about the feature:
```suggestion
    @import 'tailwindcss/utilities' important(#app);
```

If you are talking about what it does:
```suggestion
    @import 'tailwindcss/utilities' wrapper(#app);
```

Don't like either option so think `selector` is fine, but still writing this down just in case.

Reply:
`selector(…)` was Adam's initial thought but he wasn't really set on it I don't think. We looked around for possible existing discussion and the closest thing we could find was scope(…) but that will: 1. be an actual thing eventually; 2. works a bit differently.

I don't think spelling this `important(#app)` is the right move — the fact that it's the equivalent of part of the v3 important feature is _fine_ but we'd probably name it differently had that feature not existed (and still wanted to add it).

I don't mind `wrapper(#app)` though.

Reply:
@adamwathan do you have an opinion on this ☝️